e2e/tests/functional/planning/timelist.e2e.spec.js

```diff
@@ -122,3 +126,162 @@ test.describe('Time List', () => {
     });
   });
 });
+
+/**
+ * The regular expression used to parse the countdown string.
+ * Some examples of valid Countdown strings:
+ * ```
+ * '35D 02:03:04'
+ * '-1D 01:02:03'
+ * '01:02:03'
+ * '-05:06:07'
+ * ```
+ */
+const COUNTDOWN_REGEXP = /(-)?(\d+D\s)?(\d{2}):(\d{2}):(\d{2})/;
```

Comment:
i'm just going to assume that chatgpt knows that this works

Reply:
🤖